types/ember__test-helpers/index.d.ts

```diff
@@ -223,6 +224,11 @@ declare module '@ember/test-helpers/validate-error-handler' {
     export default function(callback?: (error: Error) => void): { isValid: boolean, message: string };
 }
 
+declare module '@ember/test-helpers/onerror' {
+    export function setupOnerror(handler: (error: any) => void): void;
```

Comment:
One last tweak: let’s make this `unknown` instead of `any`.

Reply:
Good call. Updated.